src/actions/bundle.ts

```
@@ -91,6 +91,19 @@ export const mulmoViewerBundle = async (context: MulmoStudioContext) => {
     });
   });
 
+  await Promise.all(
+    context.studio.script.beats.map(async (__, index) => {
+      const data = resultJson[index];
+      if (Object.keys(data.audioSources).length === 0 && data.videoSource === undefined && data.videoWithAudioSource === undefined && data.duration) {
+        const file = `silent_${index}.mp3`
```

ESLint

[error] Insert `;`

(prettier/prettier)